Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM golang:1.11-alpine3.9
 
-RUN apk update && apk add git ca-certificates
```

Comment:
We can avoid the `rm /var/cache/apk` command if we use `--no-cache`, can't we? :-)